.codeclimate.yml

```diff
@@ -23,8 +26,17 @@ plugins:
   rubocop:
     enabled: true
     channel: rubocop-0-74
+    checks:
+      Rubocop/Metrics/BlockLength:
+        enabled: false
+      Rubocop/Bundler/OrderedGems:
+        enabled: false
```

Comment:
Are these modifications extremely needy? I would rather suppress errors (if urgently needed) in this branch, but keep config the same for global PR's.

Reply:
Forgot to highlight `sql_injection` property, actually this was my main concern

Reply:
I'll leave OrderedGems here - we've already got gems in Gemfile grouped by type, not ordered by aplhabet.